moped-editor/src/views/projects/projectsListView/ProjectsListViewTable.js

```diff
@@ -230,6 +235,9 @@ const ProjectsListViewTable = ({ query, searchTerm }) => {
 
   // For each filter added to state, add a where clause in GraphQL
   // Advanced Search
+  // I tried putting useEffect around this to only update the where clause when the filters were updated
+  // however the component does not refresh to display new data
+  // useEffect(() => {
```

Comment:
I tried putting useEffect around these where/filter calls and the where/search calls. But then the component wouldn't refresh and the data was stale, so I put a pin on this for now.

Reply:
lll clean up these comments and remove them before merging, with your work on getting rid of graphql abstract, I wont be trying to fix this. thanks for checking tho